js/script.js

```diff
@@ -435,3 +429,11 @@ contactForm.addEventListener('submit', (e) => {
     error[2].parentElement.classList.add('success');
   }
 });
+
+const inputData = localStorage.getItem('mydata');
+const savedData = JSON.parse(inputData);
+window.onload = () => {
+  inputName.value = savedData.name;
+  inputEmail.value = savedData.email;
+  inputComment.value = savedData.comment;
+};
```

Comment:
- [ ] [REQUIRED] Please carefully review the edited code because I have noticed that on filling the form, no data is being stored to local storage 🤔.

![image](https://user-images.githubusercontent.com/4247740/146636257-bc3da05e-b7c5-4b38-92ca-8e7c2c8f9033.png)